2017-01-30-openwhisk-integration-with-serverless

```diff
@@ -0,0 +1,44 @@
+---
+title: Introducing OpenWhisk Integration with the Serverless Framework
+description: The new OpenWhisk provider plugin allows developers to build applications for the platform using the Serverless Framework.
```

Comment:
"For the Bluemix platform"? (Not sure if it's a platform, but it's their name for the Cloud where OpenWhisk runs on.)